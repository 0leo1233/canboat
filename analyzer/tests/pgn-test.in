#
# A list of PGNs to be processed, in order of PGN.
#
# Some are artificially changed to have non-null fields. These
# are recognisable by timestamp date 1970-01-01.
#
2011-04-25-06:25:03.603,3,129029,36,255,43,e6,f1,3a,80,9c,c6,0d,00,12,38,aa,49,eb,51,07,00,0c,44,95,fb,15,b8,00,40,e1,33,00,00,00,00,00,13,fc,09,5a,00,8c,00,ff,ff,ff,7f,00
1970-01-01T00:00:00.000Z,3,127513,61,255,8,00,c1,11,14,00,02,01,62
#SHOWBUFFERS
