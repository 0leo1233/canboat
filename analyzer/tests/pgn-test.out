{"version":"4.2.2","units":"std"}
{"timestamp":"2011-04-25-06:25:03.603","prio":3,"src":36,"dst":255,"pgn":129029,"description":"GNSS Position Data","fields":{{"SID":230,"bytes"="E6"},{"Date":"2011.04.25","bytes"="F1 3A"},{ "Time": "06:25:12","bytes"="80 9C C6 0D"},{"Latitude":52.7461333,"bytes"="00 12 38 AA 49 EB 51 07"},{"Longitude": 5.1815566,"bytes"="00 0C 44 95 FB 15 B8 00"},{"Altitude":3.400000,"bytes"="40 E1 33 00 00 00 00 00"},{"GNSS type":"GPS+SBAS/WAAS","bytes"="03","bits"="0011"},{"Method":"GNSS fix","bytes"="10","bits"="0001"},{"Integrity":"No integrity checking","bytes"="00","bits"="00"},{{"Number of SVs":9,"bytes"="09"},{"HDOP":0.90,"bytes"="5A 00"},{"PDOP":1.40,"bytes"="8C 00"},{"Geoidal Separation":null,"bytes"="FF FF FF 7F"},{"Reference Stations":0,"bytes"="00"}}}
{"timestamp":"1970-01-01T00:00:00.000Z","prio":3,"src":61,"dst":255,"pgn":127513,"description":"Battery Configuration Status","fields":{{"Instance":0,"bytes"="00"},{"Battery Type":"Gel","bytes"="01","bits"="0001"},{"Supports Equalization":"No","bytes"="00","bits"="00"},{{"Nominal Voltage":"12V","bytes"="01","bits"="0001"},{"Chemistry":"Li","bytes"="10","bits"="0001"},{"Capacity":20,"bytes"="14 00"},{"Temperature Coefficient":2,"bytes"="02"},{"Peukert Exponent":0.002,"bytes"="01"},{"Charge Efficiency Factor":98,"bytes"="62"}}}
